creator-node/src/routes/nodeSync.js

```diff
@@ -551,9 +561,11 @@ async function _nodesync (serviceRegistry, logger, walletPublicKeys, creatorNode
         throw new Error(e)
       }
     }
+    await SyncHistoryAggregator.recordSyncSuccess(logContext)
```

Comment:
no, I do not think that's necessary. this function is wrapped in a try/catch already. if it errors, the logger will log the error message and continue. https://github.com/AudiusProject/audius-protocol/pull/1516/files/682836a7944c579de3cf07e764a5616a7f0e855d#diff-63a19d6dbe036b3e0fb4521ccf41b00ffa1b12c5493223552e8293ebc364035aR43

Reply:
yeah i know, but thats not my point - what if there was a mistake in the `recordSyncSuccess()` function? that is not try-catched, the try-catch is only in the inner `recordSyncData()` function, but we are still open to errors outside of that

Reply:
? I don't understand the context of this comment. 

1. `recordSyncSuccess` calls `recordSyncData`. 
2. `recordSyncData` is wrapped in a try/catch, except when creating the logger. 
3. if `recordSyncData` were to ever fail, the try/catch will catch and log the error.
4. as a result, `recordSyncSuccess` wont ever error
5. calling `await SyncHistoryAggregator.recordSyncSuccess(logContext)` in this line here will never error

https://github.com/AudiusProject/audius-protocol/blob/682836a7944c579de3cf07e764a5616a7f0e855d/creator-node/src/snapbackSM/syncHistoryAggregator.js#L25

Reply:
yeah haha, sorry for the confusion. in current state this is definitely fine - my point is that by not try-catching at the outer-most call level, any code changes to the implementation could introduce new logic or situations that could throw an error which would be problematic since the outer caller does not try-catch. my point was that it is generally safe to wrap it at the top most level to protect against underlying implementation changes

Reply:
discussed offline, can leave as is